### PR TITLE
Add GuildManager#setFeatures

### PR DESCRIPTION
**Pull request by MinnDevelopment**
[contributing]: https://github.com/DV8FromTheWorld/JDA/wiki/5%29-Contributing

## Pull Request Etiquette

- [x] I have checked the PRs for upcoming features/bug fixes.
- [x] I have read the [contributing guidelines][contributing].

### Changes

- [ ] Internal code
- [x] Library interface (affecting end-user code) 
- [ ] Documentation
- [ ] Other: \_____ 

Closes Issue: NaN

## Description

This can be used to pause invites in a guild.

#### Example

```java
List<String> features = new ArrayList<>(guild.getFeatures());
features.add("INVITES_DISABLED");
guild.getManager().setFeatures(features).queue();
```

Supposedly this is just an experiment according to https://github.com/discord/discord-api-docs/pull/5269#issuecomment-1229050375

### CODE REVIEW

**Comment by advaith1** [comment]
https://github.com/discord/discord-api-docs/pull/5270 is relevant here

**Comment by DV8FromTheWorld** [comment]
This is definitely the bare minimum of what we could provide to do this, but it feels.. not great.
Like, regardless of the actual implementation of how discord is representing these toggles, it feels like this should be
`isInvitesPaused` and `setInvitesPaused(Boolean)`

**Comment by CheesyGamer77** [comment]
The relevant Discord Docs PR has been merged. Doesn't appear to be an experiment

File: src/main/java/net/dv8tion/jda/api/entities/Guild.java [hunk 1]
```diff
@@ -544,14 +544,27 @@ default ImageProxy getIcon()
 
     /**
      * The Features of the {@link net.dv8tion.jda.api.entities.Guild Guild}.
-     * <p>
-     * <a target="_blank" href="https://discord.com/developers/docs/resources/guild#guild-object-guild-features"><b>List of Features</b></a>
+     *
+     * <p>Features can be udpated using {@link GuildManager#setFeatures(Collection)}.
```
Comment (freya022) [inline src/main/java/net/dv8tion/jda/api/entities/Guild.java:548]:
```suggestion
     * <p>Features can be updated using {@link GuildManager#setFeatures(Collection)}.
```

File: src/main/java/net/dv8tion/jda/api/managers/GuildManager.java [hunk 2]
```diff
@@ -372,4 +379,122 @@ public interface GuildManager extends Manager<GuildManager>
     @Nonnull
     @CheckReturnValue
     GuildManager setBoostProgressBarEnabled(boolean boostProgressBarEnabled);
+
+    /**
+     * Configures the new {@link Guild#getFeatures() features} of the {@link Guild}.
+     * <br>The list of available features, including which ones can be configured is available in the
```
Comment (DV8FromTheWorld) [inline src/main/java/net/dv8tion/jda/api/managers/GuildManager.java:385]:
```suggestion
     * <br>The list of available features, including which ones can be configured, is available in the
```

**Review by DV8FromTheWorld — APPROVED** [review]
LGTM

**Pull request merged**